frontend/src/components/DataCard/index.jsx

```diff
@@ -110,6 +110,29 @@ class DataCard extends React.Component {
                                 </Button>
                             </OverlayTrigger>
                         </ButtonToolbar>
+                        {labels.length > 5 && (
+                            <div className="suggestions dropdown">
+                                <button type="button" className="btn btn-info dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                    Suggested Labels
+                                </button>
+                                <ul className="dropdown-menu dropdown-menu-right">
+                                    {labelsOptions.slice(0, 5).map(opt => (
```

Comment:
So it looks like this will always return the first 5 labels alphabetically. I know that we don't really have a model here, but I think we can start with something a bit simpler that's nevertheless useful.

I think a good starting point would be to make an api call that finds the label+descriptions with the most overlap with the text. 

There are a number of ways to do this. I think an easy one to start would be lemmatizing the text and getting the Jaccard index https://en.wikipedia.org/wiki/Jaccard_index similarity between the text and each possible label+description pair. We could do this upon data upload so the calculations only have to happen once.

Thoughts?